control/reset.py

```diff
@@ -25,15 +26,18 @@ def createReset():
                             mongodb_settings.get('port'))
     rabbitmq_conn = RabbitMQConn()
 
-    return Reset(mongo_conn, rabbitmq_conn)
+    connman = ConnmanClient()
+
+    return Reset(mongo_conn, rabbitmq_conn, connman)
 
 class Reset():
     """
     Performs reset functions on KNoT Gateway
     """
-    def __init__(self, mongo_conn, rabbitmq_conn):
+    def __init__(self, mongo_conn, rabbitmq_conn, connman):
```

Comment:
I liked the way you added these arguments.
Shows that the code is clear and easy to modify.
Good job!